README.md

```diff
@@ -35,24 +35,24 @@ These are the variables that can be passed to the role:
 | **Name** | **Default/Choices** | **Description** |
 |----------|-------------|------|
 | `device` | | specifies the path of the backing device of an encrypted device on the managed host. This device must be already configured as a LUKS device before using the role (**REQUIRED**). |
-| `passphrase` | | a valid passphrase for opening/unlocking the specified device. Recommend vault encrypting the passphrase. See https://docs.ansible.com/ansible/latest/user_guide/vault.html |
-| `key_file` | | either the absolute or relative path, on the control node, of a key file valid for opening/unlocking the specified device. |
+| `encryption_password` | | a valid password or passphrase for opening/unlocking the specified device. Recommend vault encrypting the value. See https://docs.ansible.com/ansible/latest/user_guide/vault.html |
+| `encryption_key_src` | | either the absolute or relative path, on the control node, of a file containing an encryption key valid for opening/unlocking the specified device.  The role will copy this file to the managed node(s). |
```

Comment:
Does `encryption_key_src` play an equivalent role to `encryption_password`? I.e. is `encryption_key_src: key.txt` equivalent to `encryption_password: "{{ lookup('file', 'key.txt') }}"` ?

Reply:
similar question applies to linux-system-roles/storage/pull/126, although the file there is remote, so lookup won't work.

Reply:
@sergio-correia can you please answer?

Reply:
Yeah, they are equivalent, they are just two different ways to provide a passphrase to `cryptsetup`

Reply:
Do we need to explicitly document the relationship between `encryption_password` and `encryption_key_src`?

Reply:
> Do we need to explicitly document the relationship between `encryption_password` and `encryption_key_src`?

I'll wait and see what happens with https://github.com/linux-system-roles/storage/pull/126, and I'll use similar text in this PR

Reply:
IMO we don't need to, according to the discussion in storage.

Reply:
Ok to merge?

Reply:
I am OK with it.